test/guest/openshift.setup

```diff
@@ -14,18 +18,23 @@ iptables -F
 
 dnf -y install docker
 
-# Wait for x for 5 minutes
+# Wait for x for many minutes
 function wait() {
-    for i in $(seq 1 50); do
+    for i in $(seq 1 100); do
         if eval "$@"; then
             return 0
         fi
         sleep 6
     done
+    echo "WAIT FAILED"
+    sleep 100000000
```

Comment:
Why do we want to sleep so long after a wait failing? Or is this just leftover from debugging.